src/database/seeders/20200904133732-Profile.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkInsert("Profiles", [{
+      firstName: "Olusola",
+      lastName: "Fullstack",
+      profilePicture: "http:/facebook.com",
+      userId: 1,
```

Comment:
let the userid on profile seeders be the same id as the id of users in user seeders